_dev/src/components/merchant-center-account/phone-verification/phone-verification-popin.vue

```diff
@@ -215,6 +244,42 @@ export default {
     getPhoneNumber() {
       this.phoneNumber = this.$store.getters['accounts/GET_SHOP_INFORMATIONS'].store.phone;
     },
+    resetVerificationCodeInputs() {
+      this.inputsVerificationCode = this.inputsVerificationCode
+        .map(() => ({
+          invitationId: null,
+        }));
+    },
+    goToNextInput(key) {
+      if (key.keyCode === 8) {
+        this.disableSendCodeButton = true;
+        this.isCodeValid = null;
+      }
+      const isFillCompletely = [];
+      this.inputsVerificationCode.forEach((code) => {
+        if (!code.invitationId) {
+          this.disableSendCodeButton = true;
+        } else {
+          isFillCompletely.push(code.invitationId);
+        }
+      });
```

Comment:
Then, this could be simplified with

```
this.disableSendCodeButton = !isCompletelyFilled;
```

This could be a computed by the way, this would make this method simpler, when you only manage key codes to change the focus.